examples/public/misc/piechart-vega.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CARTO.js + Category Dataviews + Vega Pie Chart</title>
+    <meta name="viewport" content="initial-scale=1.0">
+    <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600,700|Open+Sans:300,400,600" rel="stylesheet">
+    
+    <!-- Include Leaflet -->
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+    <link href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" rel="stylesheet">
+    
+    <!-- Include CARTO.js -->
+    <script src="https://libs.cartocdn.com/carto.js/v4.1.11/carto.min.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:600" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    
+    <!-- Custom Style -->
+    <link href="https://carto.com/developers/carto-js/examples/maps/public/style.css" rel="stylesheet">
+  
+    <!-- Vega -->
+    <script src="https://cdn.jsdelivr.net/npm/vega@3"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite@2"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-embed@3"></script>
+  </head>
+  <body>
+    <div id="map"></div>
+
+    <aside class="toolbox">
+      <div class="box">
+        <header>
+          <h1>Pie Chart Example</h1>
+        </header>
+        <section>
+          <p class="description open-sans">Done with Vega Lite</p>
+          <br>
+          <div id="piechart"></div>
+          <div style="margin-top: 12px;" id="legend"></div>
+        </section>
+        <footer class="js-footer"></footer>
```

Comment:
Sorry I forgot to say it in the previous round. `footer` element is not needed here since it's just used to inject the buttons to change from leaflet to gmaps when the examples have both versions and that's not the case.

Reply:
Removed!